doc/antora/modules/concepts/nav.adoc

```diff
@@ -1,15 +1,15 @@
 * xref:index.adoc[Concepts]
 ** xref:overview.adoc[Overview]
 *** xref:freeradius.adoc[What is FreeRADIUS]
-*** xref:aaa/aaa.adoc[Authentication Authorisation Accounting (AAA)]
-**** xref:aaa/authz.adoc[Authorisation]
+*** xref:aaa/aaa.adoc[Authentication Authorization Accounting (AAA)]
 **** xref:aaa/authn.adoc[Authentication]
+**** xref:aaa/authz.adoc[Authorization]
 **** xref:aaa/acct.adoc[Accounting]
 *** xref:components/architecture.adoc[RADIUS System Components]
 **** xref:components/nas.adoc[Network Access Server (NAS)]
 **** xref:components/radius_servers.adoc[RADIUS Server]
-***** xref:components/radius_servers.adoc#policy[Server Policies]
-**** xref:components/datastore.adoc[Datastores]
+***** xref:components/radius_servers.adoc#policy[Policies]
+**** xref:components/datastore.adoc[Datastore]
```

Comment:
I'm not sure why the plural has been removed?  While it's consistent with the file name, there isn't one datastore for the server, there are multiple.